extensions/net/ops_tls.rs

```diff
@@ -788,6 +796,12 @@ where
     }
   }
 
+  {
+    let mut s = state.borrow_mut();
+    let no_certificate_validation = s.borrow_mut::<NoCertificateValidation>();
+    global_no_check_certificate = no_certificate_validation.no_check_certificate.clone();
```

Comment:
The idea behind this was to give a user ability to define the excluded domains in the code and through the CLI and merge them together. Does it makes sense from security perspective?